engine/admin/index.md

```diff
@@ -159,7 +159,7 @@ you installed Docker, you may already have this file.
 7. Verify that the `docker` daemon is running as specified with the `ps` command.
 
     ```bash
-    $ ps aux | grep docker | grep -v grep
+    $ ps aux | grep [d]ocker
```

Comment:
Can you explain why you're going `grep [d]ocker` instead of `grep docker`? Isn't it the exact same thing?

And the `grep -v grep` is just removing noise from the output since without it you'll have one line like
```
docker   17370  0.0  0.1   9760  1040 pts/0    S+   22:58   0:00 grep docker
```